main.tf

```
@@ -592,3 +563,54 @@ resource "aws_config_config_rule" "iam_mfa_for_console_access" {
     "aws_lambda_permission.iam_mfa_for_console_access",
   ]
 }
+
+### RESTRICTED COMMON PORTS: ACCESS
+resource "aws_config_config_rule" "restricted_common_ports_access" {
+  count = "${var.create_config_rules && ! local.exclude_restricted_common_ports_access ? 1 : 0}"
+
+  name             = "restricted-common-ports-access"
+  description      = "Checks whether security groups that are in use disallow unrestricted incoming TCP traffic to the specified ports."
+  input_parameters = "{\"blockedPort1\":\"22\",\"blockedPort2\":\"3389\"}"
```

use a heredoc like before to make this somewhat easier to read. or could switch to jsonencode on an actual map.
```suggestion
  input_parameters = <<-EOF
    {
      "blockedPort1": "3389",
      "blockedPort2": "3389"
    }
    EOF
```